src/Http/Routing/src/EndpointRoutingMiddleware.cs

```diff
@@ -283,5 +291,8 @@ public static void MatchSkipped(ILogger logger, Endpoint endpoint)
 
         [LoggerMessage(6, LogLevel.Information, "The endpoint '{EndpointName}' is being short circuited without running additional middleware or producing a response.", EventName = "ShortCircuitedEndpoint")]
         public static partial void ShortCircuitedEndpoint(ILogger logger, Endpoint endpointName);
+
+        [LoggerMessage(7, LogLevel.Information, "Matched endpoint '{EndpointName}' is a fallback endpoint.", EventName = "FallbackMatch", SkipEnabledCheck = true)]
```

Comment:
This is only a note about how it was routed.
```suggestion
        [LoggerMessage(7, LogLevel.Debug, "Matched endpoint '{EndpointName}' is a fallback endpoint.", EventName = "FallbackMatch", SkipEnabledCheck = true)]
```